docs/_source/docs/stack_config.rst

```diff
@@ -312,49 +315,77 @@ role_arn
 * Resolvable: Yes
 * Can be inherited from StackGroup: Yes
 * Inheritance strategy: Overrides parent if set
 
+.. warning::
+   This field is deprecated as of v4.0.0 and will be removed in v5.0.0. It has been renamed to
+   `sceptre_role`_ as a clearer name for its purpose.
+
+sceptre_role
+~~~~~~~~~~~~
+* Resolvable: Yes
+* Can be inherited from StackGroup: Yes
+* Inheritance strategy: Overrides parent if set
+
 This is the IAM Role ARN that **Sceptre** should *assume* using AWS STS when executing any actions
 on the Stack.
 
-This is different from the ``role_arn`` option, which sets a CloudFormation service role for the
-stack. The ``iam_role`` configuration does not configure anything on the stack itself.
+This is different from the ``cloudformation_service_role`` option, which sets a CloudFormation
+service role for the stack. The ``sceptre_role`` configuration does not configure anything on the
+stack itself.
 
 .. warning::
 
-   If you set the value of ``iam_role`` with ``!stack_output``, that ``iam_role``
+   If you set the value of ``sceptre_role`` with ``!stack_output``, that ``sceptre_role``
    will not actually be used to obtain the stack_output, but it *WILL* be used for all subsequent stack
    actions. Therefore, it is important that the user executing the stack action have permissions to get
-   stack outputs for the stack outputting the ``iam_role``.
+   stack outputs for the stack outputting the ``sceptre_role``.
 
 For more information on this configuration, its implications, and its uses, see
-:ref:`Sceptre and IAM: iam_role <iam_role_permissions>`.
+:ref:`Sceptre and IAM: sceptre_role <sceptre_role_permissions>`.
 
 iam_role_session_duration
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 * Resolvable: No
 * Can be inherited from StackGroup: Yes
 * Inheritance strategy: Overrides parent if set
 
-This is the session duration when **Sceptre** *assumes* the **iam_role** IAM Role using AWS STS when
+.. warning::
+   This field is deprecated as of v4.0.0 and will be removed in v5.0.0. It has been renamed to
+   `sceptre_role_session_duration`_ as a clearer name for its purpose.
+
+sceptre_role_session_duration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Resolvable: No
+* Can be inherited from StackGroup: Yes
+* Inheritance strategy: Overrides parent if set
+
+This is the session duration when **Sceptre** *assumes* the **sceptre_role** IAM Role using AWS STS when
 executing any actions on the Stack.
 
 .. warning::
 
-   If you set the value of ``iam_role_session_duration`` to a number that *GREATER* than 3600, you
-   will need to make sure that the ``iam_role`` has a configuration of ``MaxSessionDuration``, and
-   its value is *GREATER* than or equal to the value of ``iam_role_session_duration``.
-
-For more information on this configuration, its implications, and its uses, see
-:ref:`Sceptre and IAM: iam_role_session_duration <iam_role_permissions>`.
+   If you set the value of ``sceptre_role_session_duration`` to a number that *GREATER* than 3600, you
```

Comment:
Looks like a vestigial "that" here. I think this is supposed to read: "to a number *GREATER* than"